packages/keyring-snap-client/src/KeyringClient.test.ts

```diff
@@ -1,10 +1,11 @@
-import {
-  type KeyringAccount,
-  type KeyringRequest,
-  type KeyringResponse,
-  KeyringClient,
-  KeyringRpcMethod,
-} from '.'; // Import from `index.ts` to test the public API
+import type {
+  KeyringAccount,
+  KeyringRequest,
+  KeyringResponse,
+} from '@metamask/keyring-api';
+import { KeyringRpcMethod } from '@metamask/keyring-api';
+
+import { KeyringClient } from '.'; // Import from `index.ts` to test the public API
```

Comment:
This is a pattern we could re-use everywhere to avoid ignoring the `index.ts` from the coverage.

Reply:
Possibly, but not sure if it adds real value to the test. If the goal is to verify that the index file exposes certain objects and types then we could create a specific test for it 🤔

Reply:
I might revamp some of our tests to properly test the public API yes (in another PR).